### DISCUSSION TITLE
Charged for GitHub Copilot even after cancelling subscription

### DISCUSSION THREAD

**Discussion by mohittgupta** [community/community#122186] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hello , 

I was charged annually even after cancelling my GitHub Copilot subscription. Please arrange a refund of $100, as I don't need this Copilot anymore. 

ticket :  https://support.github.com/ticket/personal/0/2769192

**Comment by rustamtolipov** [comment]
did you get refund?

**Reply by mohittgupta** [reply]
Yes I got the refund, after 15 days.

**Comment by chihkang** [comment]
got same issue 😡

**Reply by mohittgupta** [reply]
Raise a ticket with github support and they will help you out.
